src/types/customerInstitution.types.ts

```diff
@@ -147,3 +147,28 @@ export interface VocabularyList {
   id: string;
   vocabularies: CustomerVocabulary[];
 }
+
+export interface ClaimedChannel {
+  id: string;
+  claimedBy: {
+    id: string;
+    organizationId: string;
+  };
+  channelClaim: {
+    channel: string;
+    constraint: {
+      publishingPolicy: 'Everyone' | 'OwnerOnly';
+      editingPolicy: 'Everyone' | 'OwnerOnly';
+      scope: DegreeType[];
+    };
+  };
+}
+
+export interface ClaimedChannelList {
```

Comment:
Siden denne bare er relevant for responsen til det ene spesifikke endepunktet la jeg den typen i API-fila i min PR. Men vi har gjort forskjellig der ulike plasser, så skal ikke hevde at en løsning er bedre enn en annen. Bare til info